.github/workflows/bundling-test.yml

```diff
@@ -4,7 +4,7 @@ on:
   pull_request:
   push:
     branches:
-      - "main"
+      - "main-0.4"
```

Comment:
This is only for pushes, and generally pushing to main is a rare exception.

Reply:
Oh, isn't it for pull requests too? I just wanted the tests to run on this PR (which is to main-0.4) to make sure I didn't mess up with my backporting.

Reply:
There's nothing after `pull_request:`, which means it's not restricted. `push:` is limited to branches, branches that are called `main`. So it should run on all PRs anyway.